src/controller/index.ts

```diff
@@ -11,17 +11,29 @@ import { generateContentWithGPT } from "../utils/generateContentWithGPT";
 import { generateImage, generateImagePrompt } from "../utils/generateImage";
 import { sendMail } from "../utils/sendMail";
 import { postOnTwitter } from "../utils/postOnSocialMedia";
+import { URL } from "url";
+import Twitter from "twitter-lite";
+import { app } from "../index";
 
 interface NewsPost {
     news: string;
     emails: string[];
     from: string;
     to: string;
+    postToTwitter: boolean;
+    oauth_token?: string;
+    oauth_verifier?: string;
 }
 
+const twitterClient = new Twitter({
+    consumer_key: APP_KEY,
+    consumer_secret: APP_KEY_SECRET,
+    version: "1.1",
+});
+
 export const newAutomatedLetter = async (req: Request<{}, {}, NewsPost>, res: Response) => {
     try {
-        const { news, emails, to, from } = req.body;
+        const { news, emails, to, from, oauth_token, oauth_verifier, postToTwitter } = req.body;
```

Comment:
`oauth_token` should be renamed to `twitterOAuthToken` and `oauth_verifier` should be renamed to `twitterOAuthVerifier`